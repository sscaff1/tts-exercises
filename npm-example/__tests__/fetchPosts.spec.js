const fetchPosts = require('../fetchPosts');

describe('Fetch Posts', () => {
  it('data should have a title', async () => {
    const data = await fetchPosts();

    expect(data.title).toBeDefined();
  });

  it('should match the data snapshot', async () => {
    const data = await fetchPosts();

    expect(data).toMatchSnapshot();
  });
});
