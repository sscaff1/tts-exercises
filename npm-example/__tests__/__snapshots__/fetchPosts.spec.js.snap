// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fetch Posts should match the data snapshot 1`] = `
Object {
  "body": "test body",
  "id": 1,
  "title": "test title do not match",
  "userId": 1,
}
`;
